### PR TITLE
Disable the bonus payouts by default

### PR DESCRIPTION
**Pull request by cooperka**
Resolves https://www.pivotaltracker.com/n/projects/2476452/stories/175921513

I like to avoid commenting out code because it quickly gets stale.

⚠️ This code has not been fully tested by me personally since it takes so long to set up my infrastructure for little PRs.

Related var that changes some text on frontend: `REACT_APP_NONVOLUNTEER_PAYMENT_FEATURE`

### CODE REVIEW

File: server/app/services/triplers.js [hunk 1]
```diff
@@ -103,10 +103,12 @@ async function confirmTripler(triplerId) {
 
     // If this ambassador was once a tripler, then reward the ambassador that
     // initially claimed the then-tripler, only once per upgraded ambassador
-
     let was_once = ambassador.get("was_once");
 
-    if (was_once && !was_once.get("rewarded_previous_claimer")) {
+    // Disable the bonus payouts, at least for now.
+    let ALLOW_BONUS = false;
```
Comment (brodavi) [inline server/app/services/triplers.js]: can we make this a .env var?

**Review by brodavi — CHANGES_REQUESTED** [review]

File: server/app/services/triplers.js (hunk 1 above)
Comment (cooperka) [reply server/app/services/triplers.js]: Done! I just used the existing `first_reward_payout > 0` which probably makes it easier.

File: server/app/services/triplers.js [hunk 2]
```diff
@@ -103,14 +103,16 @@ async function confirmTripler(triplerId) {
...
     let was_once = ambassador.get("was_once");
-
-    if (was_once && !was_once.get("rewarded_previous_claimer")) {
+    if (allowBonus && was_once && !was_once.get("rewarded_previous_claimer")) {
       let was_tripler = was_once.otherNode();
       if (was_tripler.get("status") === "confirmed") {
         await was_once.update({ rewarded_previous_claimer: true });
+
+        // TODO: Should this really be dependent on the payout? It doesn't seem to be used anywhere.
         await was_tripler.update({ is_ambassador_and_has_confirmed: true });
```
Comment (cooperka) [inline server/app/services/triplers.js:114]: We should probably add a separate task to delete this field from the codebase if it's unused.

Comment (brodavi) [reply server/app/services/triplers.js:114]: +1! Thanks for finding this :)

**Comment by brodavi** [comment]
Great thanks!

**Pull request merged**